config/locales/in_person_proofing/en.yml

```diff
@@ -80,6 +80,10 @@ en:
         verify_step_enter_pii: Enter your name, date of birth, state-issued ID number,
           address and Social Security number.
       state_id:
+        alert_message: 'Your state-issued ID must not be expired. Accepted forms of ID are:'
```

Comment:
The alert shows below the hint text on the [Figma file](https://www.figma.com/file/0ANJ9EFTj1nDpjep4oB1Kc/In-Person-Proofing-(Approved-%7C-Internal)?node-id=6216%3A13786&t=urMGbUWtPHeiTKda-4), but above the hint text on the content file. I would ask Allison which one is the correct placement @JackRyan1989

Reply:
Fixed.

<img width="485" alt="Screen Shot 2023-02-16 at 11 26 35 AM" src="https://user-images.githubusercontent.com/14644234/219429081-1418dd2d-31ec-412d-9613-650328004f27.png">

Reply:
Looking good!